src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -26,32 +27,78 @@ const getParentIncident = incident => ({
 });
 
 const IncidentSplitContainer = ({ FormComponent }) => {
-  const { data, error, get, isLoading, isSuccess, post } = useFetch();
+  const { error: errorSplit, isSuccess: isSuccessSplit, post } = useFetch();
+  const {
+    data: dataParent,
+    error: errorParent,
+    get: getParent,
+    isLoading: isLoadingParent,
+    isSuccess: isSuccessParent,
+  } = useFetch();
+  const { error: errorUpdate, patch, isSuccess: isSuccessUpdate } = useFetch();
   const { id } = useParams();
   const history = useHistory();
   const dispatch = useDispatch();
-
+  const [parentIncident, setParentIncident] = useState();
+  const [directingDepartment, setDirectingDepartment] = useState([]);
+  const departments = useSelector(makeSelectDepartments);
   const subcategories = useSelector(makeSelectSubCategories);
 
   const subcategoryOptions = useMemo(
     () => subcategories?.map(category => ({ ...category, value: category.extendedName })),
     [subcategories]
   );
 
+  const updateDepartment = useCallback(
+    name => {
+      if (!name || !departments) return;
+
+      const department = departments.list.find(d => d.code === name);
+      setDirectingDepartment(department ? [{ id: department.id }] : []);
+    },
+    [departments, setDirectingDepartment]
+  );
+
   useEffect(() => {
-    get(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`);
-  }, [get, id]);
+    getParent(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`);
+  }, [getParent, id]);
 
   useEffect(() => {
-    if (isSuccess === undefined || error === undefined) return;
+    if (isLoadingParent || isSuccessParent || errorParent === undefined) return;
```

Comment:
This evaluates `isSuccessParent` as a truthy value whereas the previous line checked for a falsy value. Is this condition formulated correctly?

Reply:
Yes.